### ISSUE TITLE
Test mode

### ISSUE THREAD

**Issue by noraj1337** [IceCTF/ColdCore#25]
Is there some default accounts to test the app? Like an admin account? Is that possible to disable recaptcha in test mode (I didn't see anything in `config.py`)?

**Comment by jamiees2** [comment]
Yes, with `./ctftool users add-admin`. I don't think so, but you could hide the code here https://github.com/IceCTF/ColdCore/blob/master/templates/users/register.html 

and here:
https://github.com/IceCTF/ColdCore/blob/master/routes/users.py#L43

behind a flag

**Comment by noraj** [comment]
When logging in with `admin` is just created. I get the following error `redis.exceptions.ConnectionError: Error 111 connecting to localhost:6379. Connection refused.`. Why redis is trying to connect to a real database like in production if I am in dev mode, only dev.dn should be accessed.
